xbmc/cores/VideoPlayer/VideoPlayer.cpp

```diff
@@ -3014,6 +3014,14 @@ bool CVideoPlayer::HasRDS() const
   return m_CurrentRadioRDS.id >= 0;
 }
 
+bool CVideoPlayer::HasPVR() const
+{
+  if(m_pInputStream && m_pInputStream->IsStreamType(DVDSTREAM_TYPE_PVRMANAGER))
```

Comment:
Not 100% sure but I think this also is true for PVR recordings, not only Live TV.